src/app/Main/Body/InfoPage/Amrs/InfoPage-AmrsDialog.component.tsx

```diff
@@ -0,0 +1,132 @@
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+import {
+    makeStyles,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+} from "@material-ui/core";
+import { useTranslation } from "react-i18next";
+import { ExportButtonLabelComponent } from "../../../../Shared/Export-ButtonLabel.component";
+import { DialogComponent } from "../../../../Shared/Dialog.component";
+import { TableData } from "../InfoPage.model";
+import { AmrsTableData } from "./createAmrsTableData.service";
+
+const tableDivStyle = css`
+    margin: 1em;
+    height: inherit;
+    overflow-y: auto;
+`;
+const tableContainerStyle = css`
+    height: 100%;
+`;
+
+const tableTextStyle = css`
+    font-size: 0.5rem;
+    letter-spacing: 0;
+    white-space: nowrap;
+`;
+
+const starTextStyle = css`
+    margin-left: 1em;
+    font-size: 0.5rem;
+    letter-spacing: 0;
+`;
+
+const useStyles = makeStyles({
+    table: {
+        minWidth: 550,
+    },
+});
+
+function tableRowCellList(row: AmrsTableData): JSX.Element[] {
+    const tableCellList: JSX.Element[] = [];
+    Object.keys(row).forEach((rowKey) => {
+        tableCellList.push(
+            <TableCell
+                css={tableTextStyle}
+                component="th"
+                scope="row"
+                key={`amr-table-cell-${row.name}-${rowKey}`}

```

Comment:
Reject: row.name is not a prop of AmrsTableData

Reply:
Done. Commit "Type all amrTableData"